File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0;
        int j = 1;
        while(j < nums.size())
        {
            if(nums[j]==nums[i])
                j++;
            else
                nums[++i] = nums[j++];
            
        }
        return i+1;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0;
        j = 1;
        while(j<len(nums)):
            if(nums[j]==nums[i]):
                j=j+1;
            else:
                i = i+1
                nums[i] = nums[j];
                j =j +1
        return i+1